ws/raflow.go

```diff
@@ -434,7 +444,8 @@ type RARentableFeesDataListResponse struct {
 func SvcGetRentableFeesData(w http.ResponseWriter, r *http.Request, d *ServiceData) {
 	const funcname = "SvcGetRentableFeesData"
 	var (
-		g       RARentableFeesDataListResponse
+		g       RARentableResponse
+		rfd     RARentablesFlowData
 		foo     RARentableFeesDataRequest
 		records []RARentableFeesData
```

Comment:
You might better change the name of the variable. instead of `records`, set it something like `feesRecords`!

Reply:
ok.